e2e/README.md

```diff
@@ -46,11 +46,11 @@ dev or acceptance; typically start with your local box with this:
 export CYPRESS_BASE_URL=https://a2-dev.test
 ```
 
-Most Cypress tests also expect the following IAM-related environment variables. Set them
-according to your targeted Automate instance's IAM version.
+You must also set the IAM version, whether or not run run the flaky tests, and optionally the admin token:
```

Comment:
You have to "run"s. I think you meant "whether or not to run the flaky tests"

Reply:
Too much running ![](https://render.fineartamerica.com/images/rendered/default/print/6.000/8.000/break/images/artworkimages/medium/1/gotta-go-fast-sonic-rubick-runner.jpg)